### PR TITLE
add and update categories

### PR DESCRIPTION
**Pull request by theofficialgman**
Removed Categories (number of apps in that category):
Editors (40)

Added Categories:
Programming (19)
System Management (16)
Creative Arts (10)
CAD and Manufacturing (10)
Terminals (7)
Office (4)

New subcategory under tools:
Emulation (5)

these are just ideas. looking for feedback. 

I've considered having System Management as a subcategory under tools instead of its own primary category, same goes for Terminals. 

Programming definitely deserves its own category. Appearance has shrunk because of apps moving to better fit categories (12 apps now). 

Office and CAD and Manufacturing are on the small side, but I fee like they deserve primary categories, or otherwise put them as subcategories under tools. I wanted to move apps our of tools because it was such a massive category before, 62 apps, and shrunk to 39 apps now.

closes https://github.com/Botspot/pi-apps/issues/1446

### CODE REVIEW

**Comment by theofficialgman** [comment]
oh oops... apparently I need to make sure the github actions wiki script only runs on the master branch

File: etc/categories [hunk 1]
```diff
@@ -184,8 +184,8 @@ WhatsApp|Internet
 Windows 10 Theme|Appearance
 Windows Flasher|Tools
 Windows Screensavers|Appearance
-Wine (x86)|Tools
-WPS Office|Editors
+Wine (x86)|Tools/Emulation
```
Comment (Botspot) [inline etc/categories:187]: It's pretty sneaky of you to define an app by the exact thing its name defies. (WINE = Wine Is Not Emulator)
Not that I have a problem with this though...

**Comment by Botspot** [comment]
"CAD and Manufacturing" seems long. I think it could be replaced with "Engineering."

Other than that, this looks very well thought-out. As new types of apps are added, an occasional re-factor is good.

File: etc/categories (hunk 1 above)
Comment (theofficialgman) [reply etc/categories:187]: I know lol, but wine itself IS running under box86 which is an emulator. so I felt like it was fitting regardless

**Comment by theofficialgman** [comment]
> "CAD and Manufacturing" seems long. I think it could be replaced with "Engineering."

ah duh... why didn't I just think of that

**Comment by Botspot** [comment]
Let's merge this when #1580 is merged.

**Comment by theofficialgman** [comment]
I picked to the GUI update. hope thats ok, I'll close this

**Pull request closed without merging**